crates/host/src/wasmbus/mod.rs

```diff
@@ -1146,23 +1200,10 @@ impl ActorInstance {
                 } else {
                     resp_msg
                 };
-
-                InvocationResponse {
-                    msg: resp_msg,
-                    invocation_id: invocation.id,
-                    content_length: content_length as u64,
-                    ..Default::default()
-                }
+                Ok((resp_msg, content_length))
```

Comment:
Instead of casting `content_length`, use `try_into`/`try_from` to `u64` and fail if it does not fit